.github/workflows/tests.yml

```diff
@@ -0,0 +1,20 @@
+name: Tests
+on: [pull_request, push]
+
+jobs:
+  tests:
+    name: "Tests"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: ${{ env.ref }}
```

Comment:
These 2 lines shouldn't be here. `env.ref` is not actually defined (doesn't cause issues cause Actions just ignore it because of not being defined) - you probably took it from Red's tests workflow where we do define it because of `repository_dispatch` event.

Reply:
I may have done so :eyes: